packages/gatsby-plugin-google-analytics/src/gatsby-browser.js

```diff
@@ -1,4 +1,65 @@
+const listOfMetricsSend = new Set()
+
+function debounce(fn, timeout) {
+  let timer = null
+
+  return function (...args) {
+    if (timer) {
+      clearTimeout(timer)
+    }
+
+    timer = setTimeout(fn, timeout, ...args)
+  }
+}
```

Comment:
How often CLS and LCP can happen? Would there be chance that they trigger continuosly and hence debounce timer would reset all the time and never send it? More importantly - if that would be possible would we want to not send them?

Reply:
That's how it should work, that means your site is broken for users